src/main/kotlin/stringAddCalculator/stringAddCalculate.kt

```diff
@@ -0,0 +1,8 @@
+package stringAddCalculator
+
+fun stringAddCalculate(text: String?): Int {
+    if (text.isNullOrBlank()) return 0
```

Comment:
조건문 안에 한줄의 로직이더라도 중괄호로 묶어주면 어떨까요?

```suggestion
    if (text.isNullOrBlank()) {
        return 0
    }
```